src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -243,14 +243,15 @@
 
       var googleApiClient = [
         'gl-node/' + process.versions.node,
-        @if context.isGcloud
-          # TODO: might need to allow customizing the path to the package.json.
-          'gccl/' + require('../../package.json').version,
-        @end
-        CODE_GEN_NAME_VERSION,
+        CODE_GEN_NAME_VERSION
+      ];
+      if (opts.libName && opts.libVersion) {
```

Comment:
Nitpick: This will do the wrong thing if someone `libName` in the `opts` object but not `libVersion`. In a perfect world, if `libName` is set, then you would use it and use `options.libVersion || ''` for the version.

I do not think that is important enough to block on, though.